resources/lang/en.json

```
@@ -601,5 +601,10 @@
   "radial_menu": {
     "delete_unit_title": "Delete Unit",
     "delete_unit_description": "Click to delete the nearest unit"
+  },
+  "record": {
+    "not_found": "Record not found",
+    "error": "Failed to read record",
+    "invalid_data": "Failed to parse record data"
```

Let's stick to the "replay" terminology here to differentiate these from other record types.
```suggestion
  "replay": {
    "not_found": "Replay data not found",
    "error": "Failed to read replay data",
    "invalid": "Failed to parse replay data"
```